docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -156,6 +156,11 @@ Name                                                         Description
 ``SSLTrustStorePassword``                                    The password for the TrustStore.
 ``SSLTrustStoreType``                                        The type of the TrustStore. The default type is provided by the Java
                                                              ``keystore.type`` security property or ``jks`` if none exists.
+``SSLUseSystemTrustStore``                                   Set ``true`` to automatically use the system TrustStore based on the operating system.
+                                                             The supported OS are: ``Windows`` and ``MacOS``. For ``Windows``, the ``Windows-ROOT``
```

Comment:
change to:
The supported OSes
(plural)

Reply:
@Cromat82, can you give us a rationale for this change so that we can expand on it in docs and release notes? Java programs will use the current JVM's truststore by default unless pointed elsewhere. In what cases would a Trino site prefer to point to a system truststore? Perhaps doing this makes for smoother operation on Mac and Windows if you can integrate with the keychain apps on those systems? Perhaps this helps sites that have an in-house PKI system with a site-specific Cert Authority?

Reply:
@Ordinant sure, using the JVM truststore is fine, thing is that installing certificates there as a default option has a couple of downsides:

1) Browsers and applications which don't use the JVM won't have access to those certificates, so instead of having a single keystore with all the relevant certificates, we end up with multiple keystores throughout the system, which could be fine but could also be a hassle, depending on the situation.

2) If you rely on the default JVM truststore, you can have issues when you have multiple JREs/SDKs installed on your machine and/or when you upgrade version (e.g. certificates are not available for the JVM you are not currently using).

This is probably not so relevant for a single developer, but thinking about a scenario in which a company has to setup/upgrade an indefinite number of machines for its developers it could become an issue moving forward.

Reply:
@Ordinant let me know if you think it's enough or if I need to further expand on the subject

Reply:
Excellent explanation. Thanks!